### ISSUE TITLE
Nespresso only?

### ISSUE THREAD

**Issue by georgij** [Adyen/adyen-checkout-android#13]
Is it possible to configure this for something else than "Nespresso"?

**Comment by mickyzinho** [comment]
Yeah, why is the title of the Payment Activity "Nespresso"? Where can we set a different title, the `CheckoutRequest` class doesn't seem to offer this.

**Comment by xMikeTx** [comment]
No, you cannot change the title. I started a fork (Tractive) with the plan of adding all the things you cannot do with this library. I added the setTitle functionality feature, but there are so many other things missing, so I ended up implementing the functionality of the library by myself which is imho easier than trying to adapt this lib.

**Comment by mickyzinho** [comment]
Yeah, at least the full PaymentActivity is unusable in this state. Maybe the CreditCardForm class is useful combined with custom UI, I will check it out.
Btw. is anyone maintaining this repo? Doesn't look like anyone official cares for the issues and the last commit is from July 8.

**Comment by xMikeTx** [comment]
I don't think so, so after waiting for some response from the creator for a month, I started integrating it by myself.
I guess this is merely a "sample usage" project I guess. But they should communicate that better.
